libsolidity/interface/SMTSolverCommand.cpp

```diff
@@ -51,6 +51,22 @@ void SMTSolverCommand::setEldarica(std::optional<unsigned int> timeoutInMillisec
 		m_arguments.emplace_back("-ssol");
 }
 
+void SMTSolverCommand::setCvc5(std::optional<unsigned int> timeoutInMilliseconds)
+{
+	m_arguments.clear();
+	m_solverCmd = "cvc5";
+	if (timeoutInMilliseconds)
+	{
+		m_arguments.push_back("--tlimit-per");
+		m_arguments.push_back(std::to_string(timeoutInMilliseconds.value()));
+	}
+	else
+	{
+		m_arguments.push_back("--rlimit");
+		m_arguments.push_back(std::to_string(12000));
```

Comment:
Optionally we could define 12000 as a named constant so it's not buried here.

Reply:
Where do you suggest to put the constant?

Reply:
I admit I don't know what's the best place. First I thought about using `#define`, but I've learned it's not recommended. We can leave it like it is now.